CHANGELOG.md

```diff
@@ -42,7 +42,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Removed
 *
 ### Fixed
-* Subscribing to a message channel while unsubscribing is pending (#675)
+* Subscribing to a message channel while unsubscribing is pending (#675). This issue prevented us from subscribing to a message channel after having unsubscribed to it if no message had been sent between the un-subscription and the new subscription.
 * Using ```Visible``` instead of ```Enabled``` to make sure RNSM continues updating when off (#702)
 * Some NetworkBehaviours are disabled instead of being destroyed (#718) - This preserves the index order for NetworkBehaviours between server and clients, resulting in no indexing issue for sending/receiving RPCs.
 * Scene Bootstrapper: future proofing bootstrap scene so we don't rely on Startup's path. MTT-3707. (#735)
```

Comment:
@fernando-cortez any learnings for #742?

Reply:
Not really? It was just a pattern to reset triggers that other Actions implemented that wasn't carried over into this Action.